### ISSUE TITLE
Issues starting Docker Desktop using Windows containers

### ISSUE THREAD

**Issue by PseudotsugaMenziesii** [docker/for-win#4974]
- [x] I have tried with the latest version of my channel (Stable or Edge)
  - [X] I have uploaded Diagnostics
  - Diagnostics ID: F5451C81-1CB4-47ED-8FD8-3E010CEDB81F/20191021211047

### Expected behavior
Expected Docker service to start
### Actual behavior
Service startup eventually terminates with the tracebacks indicating that the service was unable to start.
### Information

  - Windows Version: **Windows 10 Enterprise**
  - Docker Desktop Version: **2.1.0.3**
  - Are you running inside a virtualized Windows e.g. on a cloud server or on a mac VM: **No**

### Steps to reproduce the behavior

  1. Start Docker.
  2. Examine log files to see if I can find something to paste into a Google search bar.

########
Perhaps more helpful: while trying to figure out what was going wrong, I was able to determine that the Hyper-V service wasn't running for some reason. After digging around, I was able to get the Hyper-V service to run properly after disabling CFG via the workaround outlined here: https://github.com/docker/for-win/issues/3597

That said, I'm still unable to get Docker to successfully start, and I've no clue why.

**Comment by ofek** [comment]
@PseudotsugaMeziesii If you go to `Hyper-V Manager` then on the right `Virtual Switch Manager...`, do you get an error?

**Comment by PseudotsugaMeziesii** [comment]
@ofek I do not have any errors occur when I select `Virtual Switch Manager...`. 
Just to note, I chose the 'Use Windows Containers' checkbox during the install for this issue. After submitting this issue, I reinstalled but chose NOT to use Windows containers. The Docker service has no problems starting in that situation. If I try to switch to Windows containers, it will fail in manner similar to the original problem.
